### PR TITLE
Dig through code quality

### PR DESCRIPTION
**Pull request by gpotter2**
Various cleanups/duplications fixed according to Codacy:
- `except Exception: pass` fixed
- unused variables
- variables overwriting builtins
- general cleanup

### CODE REVIEW

File: scapy/asn1/asn1.py [hunk 1]
```diff
@@ -32,6 +32,8 @@ def __init__(self, objlist=None):
         self.chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"  # noqa: E501
 
     def _fix(self, n=0):
+        # Because it's crypto, use SystemRandom (safer but ~30% slower)
```
Comment (guedou) [inline scapy/asn1/asn1.py]: I think that we don't care =)

**Comment by guedou** [comment]
@p-l- what is your call on this PR? I just approved it.

File: scapy/packet.py [hunk 2]
```diff
@@ -1390,7 +1389,7 @@ def _do_summary(self):
 
     def summary(self, intern=0):
         """Prints a one line summary of a packet."""
-        found, s, needed = self._do_summary()
+        _, s, _ = self._do_summary()
```
Comment (p-l-) [inline scapy/packet.py]: I do not agree with that change (no matter what the linter says). Using `_` is a common way to show that you don't need the results. In this specific case however, you could use directly:
```Python
         return self._do_summary()[1]
```

File: scapy/asn1/asn1.py (hunk 1 above)
Comment (p-l-) [reply scapy/asn1/asn1.py]: Agreed. We need Scapy to be fast (or at least, less slow)!

**Pull request merged**